src/components/Button.js

```diff
@@ -0,0 +1,13 @@
+/*
+ eslint-disable no-unused-vars
+*/
+
+import React from 'react';
+
+const Button = props => (
+  <button>
+    {props.btnName}
+  </button>
+);
+
```

Comment:
- [ ] Kindly add proptypes validation here. Its button name prop should be a String.